src/ArtifactoryUploader/Model/PackagesForErrorDisplay.cs

```diff
@@ -0,0 +1,46 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// SPDX-FileCopyrightText: 2024 Siemens AG
+//
+//  SPDX-License-Identifier: MIT
+// -------------------------------------------------------------------------------------------------------------------- 
+using LCT.APICommunications.Model;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LCT.ArtifactoryUploader.Model
+{
+    /// <summary>
+    /// The Model class for UnkmownPackagesAll
+    /// </summary>
+    public class PackagesForErrorDisplay
+    {
+        public List<ComponentsToArtifactory> UnknownPackagesNpm { get; set; }
+        public List<ComponentsToArtifactory> UnknownPackagesNuget { get; set; }
+        public List<ComponentsToArtifactory> UnknownPackagesConan { get; set; }
+        public List<ComponentsToArtifactory> UnknownPackagesPython { get; set; }
+        public List<ComponentsToArtifactory> UnknownPackagesDebian { get; set; }
+        public List<ComponentsToArtifactory> UnknownPackagesMaven { get; set; }
+        public List<ComponentsToArtifactory> JfrogNotFoundPackagesNpm { get; set; }
+        public List<ComponentsToArtifactory> JfrogNotFoundPackagesNuget { get; set; }
+        public List<ComponentsToArtifactory> JfrogNotFoundPackagesConan { get; set; }
+        public List<ComponentsToArtifactory> JfrogNotFoundPackagesPython { get; set; }
+        public List<ComponentsToArtifactory> JfrogNotFoundPackagesDebian { get; set; }
+        public List<ComponentsToArtifactory> JfrogNotFoundPackagesMaven { get; set; }
+        public List<ComponentsToArtifactory> JfrogFoundPackagesNpm { get; set; }
+        public List<ComponentsToArtifactory> JfrogFoundPackagesNuget { get; set; }
+        public List<ComponentsToArtifactory> JfrogFoundPackagesConan { get; set; }
+        public List<ComponentsToArtifactory> JfrogFoundPackagesPython { get; set; }
+        public List<ComponentsToArtifactory> JfrogFoundPackagesDebian { get; set; }
+        public List<ComponentsToArtifactory> JfrogFoundPackagesMaven { get; set; }
+        public List<ComponentsToArtifactory> SuccessfullPackagesNpm { get; set; }
+        public List<ComponentsToArtifactory> SuccessfullPackagesNuget { get; set; }
+        public List<ComponentsToArtifactory> SuccessfullPackagesConan { get; set; }
```

Comment:
class name says packages for error display but members are named like successful. give proper name to the class

Reply:
Updated class name